### ISSUE TITLE
exa on ubuntu 21.10 (v0.10.1) has git support disabled

### ISSUE THREAD

**Issue by pjroth** [ogham/exa#978]
When I upgraded from ubuntu from `21.04` to `21.10` and `exa --git` stopped working.  I now get the following error:
```
$ exa --git
exa: Options --git and --git-ignore can't be used because `git` feature was disabled in this build of exa
```
OS: `Ubuntu 21.10`
Version:
```
$ exa --version
exa - list files on the command-line
v0.10.1 [-git]
https://the.exa.website/
```

Thank you for this great tool!

**Comment by pjroth** [comment]
Perhaps same issue:  https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=993768

**Comment by DanBrown47** [comment]
Still not resolved  ?

**Comment by dexterfoxy** [comment]
nope!

**Comment by JSchatten** [comment]
Right now I'm using "exa" 0.9 version. Installed by cargo and added by PATH variable for bin-files (export PATH="/home/username/.cargo/bin:$PATH"):
```
sudo apt  install cargo
cargo install --vers 0.9.0 exa
```
Maybe next version (over 0.10.1) will build with git support, I'm using exa only for this.

**Comment by Antonio-Bennett** [comment]
Interestingly enough I installed latest version with cargo instead of apt and everything works fine

**Comment by ariasuni** [comment]
Yeah Ubuntu builds exa with git support disabled because of some problem they had (or at least it was the case before, not sure if it’s still the case). I’m not gonna close this right away, so that people can see this issue until it’s solved upstream, but really you need to check if it’s fixed on recent versions of Ubuntu, then complain to Ubuntu if it’s still the case, because it’s an Ubuntu issue which I know nothing about (though it would be useful if I had more info about why it was disabled in the first place…).

Anyways just install exa in another way, see e.g. here:
https://github.com/ogham/exa/issues/917#issuecomment-886219637

**Comment by Crocmagnon** [comment]
Still an issue on Ubuntu 22.04 FYI.

**Comment by seebi** [comment]
Same with Ubuntu 22.04.1 LTS ...

**Comment by JSchatten** [comment]
I've clone git repo of exa (master branch), build by cargo and git function is enable by default ($exa -Glh --git). It's all fine.

**Comment by TruongNhanNguyen** [comment]
Same problem with `--git` & `--git-ignore` on wsl2 (kernel: Ubuntu 22.04.1 LTS). When I run the command on `zsh` terminal, it prints out:

```sh
$ ~ which exa && exa --version
/usr/bin/exa
exa - list files on the command-line
v0.10.1 [-git]
https://the.exa.website/
```

I have install `exa` from Ubuntu apt store via `sudo apt install --upgrade exa`.

**Comment by tea0112** [comment]
This also happened on my PopOS.

**Comment by tdelam** [comment]
Issue on Pop_Os!

**Comment by xidameng** [comment]
I had the same issue and solved it by using cargo to install `exa` instead, somehow cargo will always re-build the binary before installing it, so the Git integration will be added during the build. 
I am using WSL2 with ubuntu 22.04 btw.

**Comment by ariasuni** [comment]
This isn’t very useful to post here information we already have (that exa on Ubuntu has git support disabled on 22.04).

This is a Debian problem (because, in short, Ubuntu uses the Debian package), I’m reposting the link:
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=993768

Please send a nice message there, or to Ubuntu, if you want to let them know this is a issue you care about (I don’t really have the time, personally).

Also, I wrote about instructions to install exa from other sources (with git enabled):
https://github.com/ogham/exa/issues/978#issuecomment-1071315254

**Comment by ryannewcomer** [comment]
hi, i got the same problem, i use sudo apt install exa and i have just remove it and install it with brew. it working now (just for someone who have the same problem)

**Comment by shalak** [comment]
@ariasuni the debian issue you linked to seems to be fixed with `0.10.1-3`, but on Ubuntu 22.04.1 there's still `0.10.1-2`.

Do we wait for ubuntu to catch up, or do we need to do something to make the bump happen?

**Comment by lecoqjacob** [comment]
If anyone is still facing this issue, just pull the repo locally, and install it from there.

`cargo install --path . --all-features`

**Comment by ariasuni** [comment]
@shalak I don’t know and don’t have the time (at least right now) to learn how Ubuntu works. exa is probably in the category of packages which never gets upgraded after release, except if it’s a security issue.

**Comment by Robert-Ernst** [comment]
Ubuntu is not ArchLinux. It doesn't have rolling upgrades. So someone has to green light it.
While it's possible to ping someone to remind them to include it in the next release, it probably take the time at least to the next release.

Here is the official package link for 22.10: 
https://packages.ubuntu.com/kinetic/exa

**Comment by pabloqpacin** [comment]
If you're like me and really want `exa --git` in Ubuntu, just [install Rust](https://www.rust-lang.org/tools/install) and its package manager. The Rust installation is pretty straightforward, defaults are good, no headache.

```bash
# Uninstall the APT package if present
sudo apt remove exa

# Install rust
curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh

# Install exa
cargo install exa
```

**Comment by Trevo525** [comment]
I believe that this issue is more specifically with Ubuntu's upstream Debian. I had the same issue with Raspbian OS which to my knowledge is based on Debian specifically.
```
$ exa --version
exa - list files on the command-line
v0.10.1 [-git]
https://the.exa.website/
```
```
$ uname -a
Linux orangepi3-lts 5.15.93-sunxi64 #23.02.2 SMP Fri Feb 17 23:48:38 UTC 2023 aarch64 aarch64 aarch64 GNU/Linux
```

Uninstalling through Apt and installing through cargo worked for me.

Thanks @pabloqpacin for the guide!

**Comment by TruongNhanNguyen** [comment]
Well, I think this topic should be closed because the solution to the problem is all given.

**Comment by mhucka** [comment]
On Ubuntu 22.04.2 LTS (and probably others), `cargo` can also be installed using `apt` or `apt-get` normally. E.g.,
```shell
apt-get install cargo
```
